### PR TITLE
[VEN-2974] Decrease XVS emissions

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN

### CODE REVIEW

File: vips/vip-410/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+    target: emission.rewardsDistributor,
+    signature: "setRewardTokenSpeeds(address[],uint256[],uint256[])",
+    params: [
+      [emission.vToken],
+      [emission.isSupplierAllocation ? speed : 0],
+      [emission.isBorrowerAllocation ? speed : 0],
+    ],
+    dstChainId: emission.chainId,
+  };
+});
```
Comment (chechu) [inline vips/vip-410/bscmainnet.ts:216]: We could reduce the commands, grouping them by destination chain and RewardsDistributor contract:
```suggestion
}).reduce((accumulator, currentValue) => {
    const command = accumulator.find(
      it => it.target == currentValue.target && it.dstChainId == currentValue.dstChainId,
    );
    if (!command) {
      accumulator.push(currentValue);
    } else {
      command.params[0].push(currentValue.params[0][0]);
      command.params[1].push(currentValue.params[1][0]);
      command.params[2].push(currentValue.params[2][0]);
    }
    return accumulator;
  }, []);
```

File: simulations/vip-410/arbitrumone.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        rewardsDistributor,
+        blocksPerMonth,
+      } of emissions) {
+        if (chainId == LzChainId.arbitrumone) {
+          const rewardDistirbutor = new ethers.Contract(rewardsDistributor, REWARDS_DISTRIBUTOR_ABI, ethers.provider);
+          if (isSupplierAllocation) {
+            expect(await rewardDistirbutor.rewardTokenSupplySpeeds(vToken)).to.equals(
+              newAllocation.div(blocksPerMonth),
+            );
+            expect(await rewardDistirbutor.rewardTokenBorrowSpeeds(vToken)).to.equals(0);
```
Comment (chechu) [inline simulations/vip-410/arbitrumone.ts:46]: I think this check should be performed when `!isBorrowerAllocation`

File: simulations/vip-410/arbitrumone.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          }
+
+          if (isBorrowerAllocation) {
+            expect(await rewardDistirbutor.rewardTokenSupplySpeeds(vToken)).to.equals(0);
```
Comment (chechu) [inline simulations/vip-410/arbitrumone.ts:50]: I think this check should be performed when `!isSupplierAllocation`

**Review by kkirka — APPROVED** [review]
Addresses, configured rates and commands LGTM

File: vips/vip-410/bscmainnet.ts [hunk 4]
```diff
@@ -0,0 +1,284 @@
...
+      signature: "setRewardTokenSpeeds(address[],uint256[],uint256[])",
+      params: [
+        [emission.vToken],
+        [emission.isSupplierAllocation ? speed : 0],
+        [emission.isBorrowerAllocation ? speed : 0],
+      ],
+      dstChainId: emission.chainId,
+    };
+  })
+  .reduce((accumulator: any, currentValue: any) => {
```
[the remark this answers is not in the document]
Comment (chechu) [reply vips/vip-410/bscmainnet.ts]: https://github.com/VenusProtocol/vips/pull/443/commits/c5e55ad0610559822017ce329672fd7a8cfb7fe7

File: vips/vip-410/bscmainnet.ts [hunk 5]
```diff
@@ -0,0 +1,284 @@
...
+export const ARBITRUM_XVS_VAULT = "0x8b79692AAB2822Be30a6382Eb04763A74752d5B4";
+export const BSC_XVS_VAULT = "0x051100480289e704d20e9DB4804837068f3f9204";
+
+export const ZKSYNC_XVS = "0xD78ABD81a3D57712a3af080dc4185b698Fe9ac5A";
+export const ETHEREYM_XVS = "0xd3CC9d8f3689B83c91b7B59cAB4946B063EB894A";
+export const ARBITRUM_XVS = "0xc1Eb7689147C81aC840d4FF0D298489fc7986d52";
+export const BSC_XVS = "0xcF6BB5389c92Bdda8a3747Ddb454cB7a64626C63";
+
+export const ZKSYNC_XVS_VAULT_REWARD = "405092592592592";
+export const ETHEREYM_XVS_VAULT_REWARD = "23333333333333333";
```
[the remark this answers is not in the document]
Comment (chechu) [reply vips/vip-410/bscmainnet.ts]: https://github.com/VenusProtocol/vips/pull/443/commits/c5e55ad0610559822017ce329672fd7a8cfb7fe7

File: vips/vip-410/bscmainnet.ts [hunk 6]
```diff
@@ -0,0 +1,284 @@
...
+export const BSC_BLOCKS_PER_MONTH = 864000;
+
+export const ZKSYNC_XVS_REWARDS_DISTRIBUTOR = "0x7C7846A74AB38A8d554Bc5f7652eCf8Efb58c894";
+export const ETHEREUM_XVS_REWARDS_DISTRIBUTOR_CORE = "0x886767B62C7ACD601672607373048FFD96Cf27B2";
+export const ETHEREUM_XVS_REWARDS_DISTRIBUTOR_LST = "0x1e25CF968f12850003Db17E0Dba32108509C4359";
+export const ARBITRUM_XVS_REWARDS_DISTRIBUTOR_CORE = "0x53b488baA4052094495b6De9E5505FE1Ee3EAc7a";
+export const ARBITRUM_XVS_REWARDS_DISTRIBUTOR_LST = "0x6204Bae72dE568384Ca4dA91735dc343a0C7bD6D";
+
+export const ZKSYNC_XVS_VAULT = "0xbbB3C88192a5B0DB759229BeF49DcD1f168F326F";
+export const ETHEREYM_XVS_VAULT = "0xA0882C2D5DF29233A092d2887A258C2b90e9b994";
```
[the remark this answers is not in the document]
Comment (chechu) [reply vips/vip-410/bscmainnet.ts]: https://github.com/VenusProtocol/vips/pull/443/commits/c5e55ad0610559822017ce329672fd7a8cfb7fe7

File: vips/vip-410/bscmainnet.ts [hunk 7]
```diff
@@ -0,0 +1,284 @@
...
+export const ARBITRUM_XVS_VAULT = "0x8b79692AAB2822Be30a6382Eb04763A74752d5B4";
+export const BSC_XVS_VAULT = "0x051100480289e704d20e9DB4804837068f3f9204";
+
+export const ZKSYNC_XVS = "0xD78ABD81a3D57712a3af080dc4185b698Fe9ac5A";
+export const ETHEREYM_XVS = "0xd3CC9d8f3689B83c91b7B59cAB4946B063EB894A";
```
[the remark this answers is not in the document]
Comment (chechu) [reply vips/vip-410/bscmainnet.ts]: https://github.com/VenusProtocol/vips/pull/443/commits/c5e55ad0610559822017ce329672fd7a8cfb7fe7

**Review by chechu — APPROVED** [review]

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/410?chainId=56

**Pull request merged**